docs/custom-meta-boxes.md

```diff
@@ -0,0 +1,69 @@
+# Custom Meta Boxes
+
+It's common practice to need to extend the CMS admin in a variety of ways, especially when creating custom post types or adding new admin pages.
+
+HM Platform provides the [CMB2](https://github.com/cmb2/cmb2) framework to facilitate the rapid development of this functionality without reinventing the wheel.
+
+The framework allows you to:
+
+- Add custom meta boxes for post types
+- Add custom meta boxes for taxonomy terms
+- Add custom options pages
+- Add custom user profile fields
+- Add custom forms on the front end
+- Register custom data for the REST API
+- Create new custom field types
+
+There is a [large eco-system of custom field types](https://github.com/CMB2/CMB2#custom-field-types) available as well.
```

Comment:
I'm wondering if we could list out all the first-party field types?

Reply:
Expanding a bit on that, maybe we could take https://github.com/CMB2/CMB2/wiki/Field-Types and put it in our docs, just thinking of maybe trying out how deep we want to do in the docs, this could be a use/test case.

Reply:
Do you mean the entire page?

Reply:
I'm not sure because it links off to a bunch of other related wiki pages. So where does it stop?